docs/modules/globalfee.md

```diff
@@ -200,10 +200,9 @@ Note that the required amount of `uatom` in globalfee is overwritten by the amou
   
 ### Case 7
 
-**Setting:** globalfee=[0.1uatom], minimum-gas-prices=0.2uatom,1stake, gas=200000, bypass-min-fee-msg-types = ["/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward"]
+**Setting:** globalfee=[0.1uatom], minimum-gas-prices=[0.2uatom, 1stake], gas=200000, bypass-min-fee-msg-types = ["/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward"]
 
-Note that the required amount of `uatom` in globalfee is overwritten by the amount in minimum-gas-prices. 
-Also, the `1stake` in minimum-gas-prices is ignored.
+Note that the required amounts of `uatom` and `stake` in minimum-gas-prices is are ignored.
```

Comment:
@yaruwangway did I get it right?

Reply:
"1stake" is ignore due to the denom is not in globalfee fee
 "uatom" is not ignored, but rather when combining global fee, it is being "merged", so globalfee provides 0.1uatom, while minimum-gas-prices wants 0.2 uatom, so 0.2 uatom is taken to check the paid fees.